configuration/igniteVersionToDisplayName.properties

```diff
@@ -0,0 +1,2 @@
+1.2.9.fuse-000001-redhat-1=1.2.9.fuse-000001-redhat-1 (Fuse Ignite TP3)
```

Comment:
isn't Fuse Online finally the chosen name? https://www.redhat.com/en/explore/fuse-online

Reply:
I created https://issues.jboss.org/browse/FUSETOOLS-2854 as discussed on chat to let change name if needed after the work on the wizard refactoring

Reply:
thanks